tool_nodes/dora-rerun/src/main.rs

```diff
@@ -21,87 +21,94 @@ fn main() -> Result<()> {
     let mut options = SpawnOptions::default();
 
     let memory_limit = std::env::var("RERUN_MEMORY_LIMIT")
-        .context("Could not read image height")?
+        .context("Could not read image height")
+        .unwrap_or("25%".into())
```

Comment:
I think we should handle `VarError::NotPresent` differently than `VarError::NotUnicode`. In the former case, it's fine to provide a default, but the latter should result in an error IMO.

Reply:
That makes sense. I'll do the same for the rest of the env variables.